sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -740,6 +784,12 @@ export interface DeleteOperationInput {
     partitionKey?: PartitionKey;
 }
 
+// @public
+export interface EpkRange {
+    maxExclusive: string;
```

Comment:
These are internal in .NET and Java, should they be also internal here?

Reply:
We are planning to expose this for users to pass in custom range.

Reply:
How can users construct a custom range? These values are hashes, unless we expose the internal hashing algorithm, which varies from collection to collection (V1 != V2), how would users construct this?

Reply:
Thanks for this, I have made changes to block custom ranges.